src/Microsoft.OpenApi.Hidi/Microsoft.OpenApi.Hidi.csproj

```diff
@@ -15,7 +15,7 @@
     <PackageId>Microsoft.OpenApi.Hidi</PackageId>
     <ToolCommandName>hidi</ToolCommandName>
     <PackageOutputPath>./../../artifacts</PackageOutputPath>
-    <Version>1.0.0-preview10</Version>
+    <Version>1.1.0-preview1</Version>
```

Comment:
Why not just jump to `1.0.0-preview11` ?

Reply:
because don't it will solve the issue, it seems that nuget does an alpha comparison as opposed to considering the numbers by groups.

Reply:
Okay.